text/logs/0114-log-correlation.md

```diff
@@ -0,0 +1,137 @@
+# Conventions for Trace and Resource Association in Logs
+
+To set out standards for correlating traces and resources in existing logs
+
+## Motivation
+
+In order to correlate log entries with the tracing spans that were occurring 
+when the log entry was created, we need to emit certain information in the logs 
+to correlate the data entries. This document attempts to lay out the information 
+that needs to be emitted, and sets forth several approaches for doing so.
+
+## Explanation
+
+Two types of correlation need to happen to tie a log record into its full 
+execution context. The first is Request Correlation, which ties the record to 
+operations that were occurring when the record was created. The second is 
+Resource Correlation, which associates the entry with where the event occurred 
+such as a host, a pod, or a virtual machine.
+
+### Request Correlation
+
+Trace correlation is achieved primarily with two values- a trace identifier and 
```

Comment:
Are "Trace correlation" and "Request correlation" interchangeable and equivalent terms?

Reply:
I don't have an answer, no preference here. Not sure if have a standardized terminology for this already. Perhaps check the rest of the spec and see if there is one.
If there is none let's choose one and stick with it.